.github/workflows/build.yml

```diff
@@ -8,6 +8,10 @@ on:
   pull_request:
     branches: main
 
+concurrency:
```

Comment:
Could you add a few comments to describe what these changes do?

Reply:
Done